src/CarbonAware.DataSources/CarbonAware.DataSources.ElectricityMapsFree/src/CarbonAware.DataSources.ElectricityMapsFree.csproj

```diff
@@ -0,0 +1,30 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <InternalsVisibleTo Include="CarbonAware.DataSources.ElectricityMapsFree.Tests" />
+    <InternalsVisibleTo Include="CarbonAware.DataSources.ElectricityMapsFree.Mocks" />
+    <InternalsVisibleTo Include="CarbonAware.DataSources.Registration" />
+    <InternalsVisibleTo Include="DynamicProxyGenAssembly2" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\CarbonAware\src\CarbonAware.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="7.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Http" />
```

Comment:
I'd add a version requirement here

Reply:
I'm not sure how the correct version can be found. I've seen that other projects (GSF.CarbonAware and CarbonAware.DataSources.WattTime) use version 6.0.0, so I've picked that. By the way, ElectricityMaps (the non-free one) doesn't set it either, so I've set it for that project too. Maybe it isn't very clean to fix EM in a PR about EMFree, but creating a separate issue, branch and pull request just for this seems really overkill.